packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/index.test.ts

```diff
@@ -177,267 +175,5 @@ describe('Telemetry', () => {
       assert.notCalled(mShowInformation);
       expect(teleStub.firstCall.args).to.eql([true]);
     });
-
-    xit('Should send telemetry data', async () => {
-      // create vscode extensionContext in which telemetry msg has been previously shown
-      mockExtensionContext = new MockExtensionContext(true);
```

Comment:
Does the MockExtensionContext get tested elsewhere? Also, is the MockExtensionContext something we're going to continue to care about with the jest updates?

Reply:
Nope...This is uningration testing at it's finest.  We have tests in vscode-integration land that call methods directly with a mocked extension context.  The MockExtensionContext is a testing artifact so there are no tests to test the test.  But's it's def a big ol code smell that we have this large class that we just use for testing.